typescript/vrotsc/src/compiler/transformer/fileTransformers/workflow.ts

```diff
@@ -515,10 +522,39 @@ export function getWorkflowTransformer(file: FileDescriptor, context: FileTransf
 
 		function buildAttributes(attributes: WorkflowParameter[]): void {
 			attributes.forEach(att => {
-				stringBuilder.append(`<attrib name="${att.name}" type="${att.type}" read-only="false" />`).appendLine();
+				stringBuilder.append(`<attrib name="${att.name}" type="${att.type}" read-only="false"`);
+				buildBindAttribute(att)
+				stringBuilder.append(` />`).appendLine();
 			});
 		}
 
+		function buildBindAttribute(att: WorkflowParameter) {
+			let value = "" + att.defaultValue;
+			if (att.bind) {
+				const index = value.lastIndexOf("/");
+				if (index == -1) {
+					throw new Error(`Invalid syntax for attribute "${att.name}" in workflow "${workflow.name}". It is specified that this value is bound to a Configuration (Elemene) but its value "${value}" `
+						+	`cannot be mapped to a Configuraiton Element path. No / separator in value. Expected formaat is "/Path/To/Confif/variable".`);
+				}
+				if (index >= value.length-1) {
+					throw new Error(`Invalid syntax for attribute "${att.name}" in workflow "${workflow.name}". It is specified that this value is bound to a Configuration (Elemene) `
+						+	`but its value "${value}" ends in / character. Expected formaat is "/Path/To/Confif/variable".`);
+				}
+				const key = value.substring(index+1).trim();
+				const path = value.substring(0, index).trim();
+				const id = context.configIdsMap[path];
+				if (id == null) {
+					throw new Error(`Invalid syntax for attribute "${att.name}" in workflow "${workflow.name}". It is specified that its value is bound to configuration element with path "${path}" and variable "${key}"`
+						+	`, but a configuration element with path "${path}" cannot be found in project at that stage. If you belive it is indeed really part of the project, `
+						+	`please try moving the file earlier alphabetically so it is processed earlier than the workflow that uses it. Currently available configuration elements are: `
```

Comment:
Good catch. Maybe is good idea to have feature task to extend processing of the workflows to be after configurations and resource elements - to avoid such errors.